great-idea/css/index.css

```diff
@@ -63,4 +63,182 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 880px;
+	margin: 0 auto;
+  }
+
+/* ------   Navigation   ------ */
+
+header{
+	display:flex;
+}
+header nav {
+	width: 691px;
+	vertical-align: top;
+	margin-top: 35px;
+  }
+  
+  header nav a {
+	margin-right: 65px;
+	font-size: 14px;
+	color: gray;
+	text-decoration: none;
+  }
+  
+  header nav a:hover {
+	text-decoration: underline;
+  }
+  
+  header nav a:first-child {
+	margin-left: 0;
+  }
+  
+  header nav a:last-child {
+	margin-right: 0;
+  }
+  
+  header .logo {
+	margin-top: 25px;
+	display:flex;
+  }
+ 
+  /* ------   Header   ------ */
+
+  .top-content{
+	  display:flex;
+  }
+  
+  .top-content .tc-text{
+	vertical-align: top;
+	margin-top:120px;
+	text-align:center;
+	font-size: 80px;
+	padding:0px 75px;
+  }
+  
+  button{
+	font-size: 17px;
+	padding:10px 40px;
+	border: 1px solid black;  
+  }
+
+  .top-content img {
+	margin-top: 70px;
+  }
+
+	/* ------   Main Content   ------ */
+
+  .main-content {
+	border-top: 1px solid black;
+	margin-top: 75px;
+	padding-top: 40px;
+	
+  }
+
+  .main-content .first-sec {
+	padding: 0 10px;
+	display:flex;
+  }
+  
+  .main-content .first-sec .text-content {
+	width: 410px;
+	margin-right: 35px;
+	margin-bottom: 30px;
+  }
+
+  .main-content .first-sec .text-content:last-child {
+	margin-right: 0;
+  }
+  .main-content .second-sec {
+	margin-top: 35px;
+	padding: 0 10px;
+	display:flex;
+  }
+  
+  .main-content .second-sec .text-content {
+	width: 275px;
+	margin-right: 12px;
+  }
+  
+  .main-content .second-sec .text-content:last-child {
+	margin-right: 0;
+  }
+  
+    /* ------   Contact & Footer   ------ */
+
+  .contact {
+	margin-top: 60px;
+	border-top: 1px solid black;
+	padding: 40px 10px 0;
+  }
+  
+  .contact p {
+	margin: 20px 0;
+  }
+  
+  footer {
+	margin: 35px 0;
+	text-align: center;
+	font-size: 14px;
+  }
+
+  /* ----------- Services Page ------------- */
+
+  header{
+	margin-bottom:15px;
+  }
+
+  .services{
+	  margin:30px 0 30px 0;
+	  padding-bottom: 30px;
+	  border-bottom: 1px solid black;
+  }
+
+  .services h1{
+	  font-size: 25px;
+  }
+
+
+  /* ----------- Boxes ------------- */
+
+.boxes{
+	display:flex;
+	flex-wrap: wrap;
+	align-content: space-between;
+	border-bottom:1px solid black;
+}
+
+.boxes div{
+border:1px solid black;
+padding:20px;
+width:45%;
+margin:0 auto;
```

Comment:
In order to have the margin of the boxes flushed with the sections above and below it, you can add `justify-content: space-between;` on the `.boxes` parent div, and remove the margin property here. Give it a try and tell me what you think!